test/unit/processing/delta/reallocate-to-ledger.test.js

```diff
@@ -11,7 +11,7 @@ describe('allocate to ledgers', () => {
     expect(updatedPaymentRequests[0].ledger).toBe(AR)
   })
 
-  test('should reallocate all AR to AP if unsettled value is greater than current request recovery', () => {
```

Comment:
Should this description match the one above but to `all AR to AP`? It's describing a valid scenario in which this switch will happen but the functionality doesn't look/compute so is a bit of a red heron

Reply:
Fair point.  I actually think the above test is named incorrectly too.

The intention of the test is to see if the ledger changes.  The reason why the ledger changes is outside the scope so no need to mention it and the reasons are actually covered in higher level tests already.  I've therefore renamed both these tests.